app/controllers/settings/preferences_controller.rb

```diff
@@ -36,6 +37,6 @@ def update
   private
 
   def user_params
-    params.require(:user).permit(:locale, :setting_default_privacy, :setting_boost_modal, :setting_auto_play_gif, notification_emails: [:follow, :follow_request, :reblog, :favourite, :mention, :digest], interactions: [:must_be_follower, :must_be_following])
+    params.require(:user).permit(:locale, :setting_default_privacy, :setting_boost_modal, :setting_auto_play_gif, :setting_site_theme, notification_emails: [:follow, :follow_request, :reblog, :favourite, :mention, :digest], interactions: [:must_be_follower, :must_be_following])
```

Comment:
it looks like this part has changed since v1.1.2 in `upstream/master`, so there will be some merge conflicts when I upgrade to the next release